test/schema/me/lot_standing.js

```diff
@@ -0,0 +1,153 @@
+import sinon from 'sinon';
+import { graphql } from 'graphql';
+import schema from '../../../schema';
+
+describe('LotStanding type', () => {
```

Comment:
This spec is basically the same as `BidderStatus`, ideally so one can replace the other :)